src/teleop/gui/src/components/AutonWaypointItem.vue

```diff
@@ -59,7 +60,29 @@ export default {
 
     sec_enabled: function() {
       return this.odom_format == 'DMS';
+    },
+
+    output: function() {
+      switch(this.odom_format) {
+        case "D":
+          return {
+            lat: convertDMS({d: this.waypoint.lat, m: 0, s: 0}, "D"),
+            lon: convertDMS({d: this.waypoint.lon, m: 0, s: 0}, "D")
+          }
+        case "DM":
+          return {
+            lat: convertDMS({d: this.waypoint.lat, m: 0, s: 0}, "DM"),
+            lon: convertDMS({d: this.waypoint.lon, m: 0, s: 0}, "DM")
+          }
+        case "DMS":
+          return {
+            lat: convertDMS({d: this.waypoint.lat, m: 0, s: 0}, "DMS"),
+            lon: convertDMS({d: this.waypoint.lon, m: 0, s: 0}, "DMS")
```

Comment:
This seems a little silly... Is there something wrong with just passing `this.odom_format` to `convertDMS` and getting rid of all of this duplication?